### DISCUSSION TITLE
モバイル学習コンテンツで、Expoの説明がほしい

### DISCUSSION THREAD

**Discussion by nagano-ryo** [ws-4020/mobile-app-crib-notes#258] [Ideas]
学習コンテンツFBより

https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/getting-started/create-project
プロジェクトの作成

Expoの説明やリンクがほしいところ。
なお、次項の「アプリの実行」にExpo Goについての説明がある。

https://ws-4020.github.io/mobile-app-crib-notes/react-native/learn/getting-started/launch-created-app
アプリの実行

**Comment by creature-water-valley** [comment]
以下のPRで対応済み
- https://github.com/ws-4020/mobile-app-crib-notes/pull/1294/files#diff-7cee4c08fad0f50ffd815de15cd1de0339dac22d6cb29538669acdf15384802aR1
